lib/ansible/modules/system/pamd.py

```diff
@@ -405,15 +405,21 @@ def add_module_arguments(service, old_rule, module_args):
     result = {'action': 'args_present'}
     changed = False
     change_count = 0
-    if isinstance(module_args, ansible.module_utils.six.string_types):
+    if isinstance(module_args, string_types):
         module_args = module_args.split(' ')
 
     for rule in service.rules:
         if (old_rule.rule_type == rule.rule_type and
                 old_rule.rule_control == rule.rule_control and
                 old_rule.rule_module_path == rule.rule_module_path):
             for arg_to_add in module_args:
```

Comment:
This is pre-existing in the code but there should be a check whether module_args is None otherwise this loop will traceback.  I think it should be fixed in the argument_spec, I'll make a note there.